feature-libs/cart/saved-cart/components/list/saved-cart-list.component.html

```diff
@@ -0,0 +1,139 @@
+<ng-container *ngIf="savedCarts$ | async as savedCarts">
+  <ng-container *ngIf="!(isLoading$ | async); else loading">
+    <div class="cx-saved-cart-list-header">
+      <h3>
+        {{
+          'savedCartList.savedCarts' | cxTranslate: { count: savedCarts.length }
+        }}
+      </h3>
+    </div>
+
+    <ng-container *ngIf="savedCarts?.length > 0; else noSavedCarts">
+      <table class="table cx-saved-cart-list-table">
```

Comment:
Why we didn't used cx-table component here?

Reply:
The reason is it's not my-company (even though it's in the shared lib)

It wouldn't match what we have currently for other listing pages that is not my-company.
I believe we can re-work on this with a ticket that changes all the other listing pages that is not my-company in whole ticket in the future. 
Main reason: to normalize with other listing pages that is not 'my-company'

Reply:
will keep that in mind